packages/admin/cms-admin/src/dam/DamPage.tsx

```diff
@@ -1,8 +1,56 @@
+import { messages } from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
 import * as React from "react";
+import { FormattedMessage } from "react-intl";
+import { useRouteMatch } from "react-router";
 
+import { ContentScopeIndicator } from "../contentScope/ContentScopeIndicator";
+import { ContentScopeInterface, useContentScope } from "../contentScope/Provider";
+import { useContentScopeConfig } from "../contentScope/useContentScopeConfig";
+import { DamScopeProvider } from "./config/DamScopeProvider";
 import { DamTable } from "./DamTable";
-function DamPage(): React.ReactElement {
-    return <DamTable />;
+
+type Props = {
+    renderContentScopeIndicator?: (scope: ContentScopeInterface) => React.ReactNode;
+};
+
+const defaultRenderContentScopeIndicator = () => (
```

Comment:
I've added this default indicator for global content when no scope is used to ease upgrading. Should we do that?

Reply:
yes, good idea. many apps will have a global dam.